onebusaway-android/src/main/java/org/onebusaway/android/ui/TripPlanFragment.java

```diff
@@ -70,6 +70,10 @@
 
 public class TripPlanFragment extends Fragment {
 
+    interface Host {
```

Comment:
From a quick look at the code, it seems that this is the callback from the fragment to the Activity that indicates the fields have been populated and a trip plan can occur?

If so, I'd prefer you change this to `interface Listener {`, with a method `void requestTripPlan();`

Please also add Javadocs to indicate what it is and what the method does.